src/Okta.Sdk/Generated/User.Generated.cs

```diff
@@ -14,7 +14,9 @@
 namespace Okta.Sdk
 {
     /// <inheritdoc/>
+    [ResourceObject(NullValueHandling = ResourceNullValueHandling.Include)] 
     public sealed partial class User : Resource, IUser
+    
```

Comment:
Did the template change to add a space here?

Reply:
yes, may be we need to check the template. The same line is added for example in `GroupSchema.Generated.cs` which is also using the the same errata rule

Reply:
I believe the  [ResourceObject(NullValueHandling = ResourceNullValueHandling.Include)]  attribute should applied to UserProfile instead of user.

Reply:
not really, we're talking about this endpoint https://developer.okta.com/docs/reference/api/users/#update-user
Setting the attribute for `User` will affect UserProfile as well, but setting it for UserProfile will not have effect when updating User object